.github/workflows/builds.yml

```diff
@@ -398,13 +398,13 @@ jobs:
           -DACTS_USE_SYSTEM_EIGEN3=OFF
           -DACTS_BUILD_PLUGIN_JSON=ON
       - name: Build
-        run: ${SETUP} && ( cmake --build build -- || cmake --build build -- )
+        run: ${SETUP} && cmake --build build
```

Comment:
I believe that second try was actually intentional by @paulgessinger because we saw some random failures?

Reply:
Yes, this is to implement sort of retry mechanism because this build fails for technical reasons. Can you revert this bit?

Reply:
I'll do so, and add a comment about why this is necessary too.